### PR TITLE
LOVE-1345 - Document the upgrade for a single node deployment for XLD / XLR

### PR DESCRIPTION
**Pull request by ielshal**
## Definition of Done

**General**
 - [ ] Branch is up-to-date with master
 - [ ] Code is reviewed and tested by someone in the Kube team
 - [ ] If there are user facing changes (new env variables for example) update the docs and make sure to notify the docs team to update official documentation

### CODE REVIEW

File: docs/upgrade-single-node-deployment.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+If you want to upgrade a single node instance of XL Release or XL Deploy using Docker follow these steps,
+
+1. Make sure to read our docs for upgrade and release notes for each version you want to upgrade to.
+2. Stop current running docker container which have old version ythat need to be upgraded.
+3. Start new docker container with new tag which represent version you want to upgrade to, taking in consideration below
+    * include all volumes already being used with old docker images
```
Comment (eltonkevani) [inline docs/upgrade-single-node-deployment.md]: Need a note here that he cannot just mount the old conf volume, he has to modify the some conf files for the new version wheen applicable

Comment (ielshal) [reply docs/upgrade-single-node-deployment.md]: Okay will add this .. however i alreaday mentioned below,
Make sure to read our docs for upgrade and release notes for each version you want to upgrade to.

Comment (eltonkevani) [reply docs/upgrade-single-node-deployment.md]: yeah just be more specific so we do not miss it

Comment (ielshal) [reply docs/upgrade-single-node-deployment.md]: yep done

**Pull request merged**